### PR TITLE
Wikipedia bug fix

### PR DESCRIPTION
**Pull request by samiyaakhtar**
- Fixed the issue with some wikipedia pages showing the wrong data

### CODE REVIEW

File: client/scraper/wikipedia_metadata.py [hunk 1]
```diff
@@ -11,13 +10,13 @@ class WikipediaMetadata(Metadata):
...
 
     def parse_content(self, response):
-        document = lxml.html.fromstring(response.content)
-        title = document.find(".//title").text
+        m = re.search(r"wiki\/(\S+)", response.url)
+        title = m.group(1)
 
         page = wikipedia.page(title)
         self.prop_map["description"] = page.summary
         self.prop_map["title"] = title
```
Comment (MeenuV) [inline client/scraper/wikipedia_metadata.py]: I think that the title should still be something like "Starbucks - Wikipedia the free encyclopedia" or something like that. Maybe we should fetch it from both places?

**Comment by MeenuV** [comment]
@samiyaakhtar can you add tests for this?

File: client/scraper/wikipedia_metadata.py [hunk 2]
```diff
@@ -11,13 +10,14 @@ class WikipediaMetadata(Metadata):
...
+        title = m.group(1)
         document = lxml.html.fromstring(response.content)
-        title = document.find(".//title").text
 
         page = wikipedia.page(title)
         self.prop_map["description"] = page.summary
-        self.prop_map["title"] = title
-        self.prop_map["image"] = page.images[0]
+        self.prop_map["title"] = document.find(".//title").text
+        self.prop_map["images"] = page.images[0:10] if len(page.images) > 9 else page.images
```
Comment (MeenuV) [inline client/scraper/wikipedia_metadata.py]: If `[0:10]` is index 10 inclusive, then should it be checking `[0:9]` instead?

Comment (samiyaakhtar) [inline client/scraper/wikipedia_metadata.py]: [0:10] is not 10 inclusive. It includes 0-9

Comment (MeenuV) [inline client/scraper/wikipedia_metadata.py]: ok cool

**Comment by MeenuV** [comment]
LGTM :+1:

**Pull request merged**